spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -432,239 +425,252 @@ dataflow:>task list
 
 === TriggerTask
 
-One way to launch a task using the `task-launcher` is to use the https://github.com/spring-cloud-stream-app-starters/triggertask/blob/v1.2.0.RELEASE/spring-cloud-starter-stream-source-triggertask/README.adoc[triggertask] source. The `triggertask` source
-will emit a message with a `TaskLaunchRequest` object containing the required launch information.
-The `triggertask` can be added to the available sources by executing the app register command as follows (for the Rabbit Binder):
+One way to launch a task with the `task-launcher` is to use the https://github.com/spring-cloud-stream-app-starters/triggertask/blob/v1.2.0.RELEASE/spring-cloud-starter-stream-source-triggertask/README.adoc[triggertask] source.
+The `triggertask` source emits a message with a `TaskLaunchRequest` object that contains the required launch information.
+The `triggertask` can be added to the available sources by running the app register command, as follows (for the Rabbit Binder, in this case):
 
-```
-app register --type source --name triggertask --uri maven://org.springframework.cloud.stream.app:triggertask-source-rabbit:1.2.0.RELEASE
-```
+`app register --type source --name triggertask --uri maven://org.springframework.cloud.stream.app:triggertask-source-rabbit:1.2.0.RELEASE`
 
-An example of this would be to launch the timestamp task once every 60 seconds, the stream to implement this would look like:
+For example, to launch the timestamp task once every 60 seconds, the stream implementation would be as follows:
 
-```
+[source]
 stream create foo --definition "triggertask --triggertask.uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.2.0.RELEASE --trigger.fixed-delay=60 --triggertask.environment-properties=spring.datasource.url=jdbc:h2:tcp://localhost:19092/mem:dataflow,spring.datasource.username=sa | task-launcher-local --maven.remote-repositories.repo1.url=http://repo.spring.io/libs-release" --deploy
-```
 
-If you execute `runtime apps` you can find the log file for the task launcher sink. Tailing that file you can find the log file for the launched tasks. The setting of `triggertask.environment-properties` is so that all the task executions can be collected in the same H2 database used in the local version of the Data Flow Server.  You can then see the list of task executions using the shell command `task execution list`
+If you run `runtime apps`, you can find the log file for the task launcher sink.
+By using the `tail` command on that file, you can find the log file for the launched tasks.
+Setting of `triggertask.environment-properties` lets all the task executions can be collected in the same H2 database used in the local version of the Data Flow Server.
```

Comment:
this line is a bit unclear.  Possibly: "Setting of `triggertask.environment-properties` establishes the Data Flow Server's H2 Database as the database where the task executions will be recorded."